src/challenge/lock-logic.js

```diff
@@ -25,6 +25,18 @@ function changeDialValue (index, incrementBy) {
   // call the redirect() function with your name
   // eg: redirect('larry-lobster')
   // the redirect function will only redirect if the lockState is unlocked
+  if (lockState.wheels[index] + incrementBy > 9) {
+    lockState.wheels[index] = 0
+  } else if (lockState.wheels[index] + incrementBy < 0) {
+    lockState.wheels[index] = 9
+  } else {
+    lockState.wheels[index] += incrementBy
+  }
+  if (lockState.wheels[0] === SECRET_COMBO[0] & lockState.wheels[1] === SECRET_COMBO[1] &
```

Comment:
Please remove the dependency on the lock's length

Reply:
> Please remove the dependency on the lock's length

Removed dependency on specific lock length and made the assumption that the secret combo length will always be equal to the lock's length

Reply:
💯